webui/react/src/types.ts

```diff
@@ -12,9 +12,17 @@ export type NullOrUndefined<T = undefined> = T | null | undefined;
 export type Point = { x: number; y: number };
 export type Range<T = Primitive> = [T, T];
 export type Eventually<T> = T | Promise<T>;
+
+// DEPRECATED
 /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
 export type RawJson = Record<string, any>;
```

Comment:
question: do we also want to replace `RawJson` with `Json` in a later task?

Reply:
I took a stab at it and most of the places using this aren't actually using it as Json, just as an untyped object. In reality those places need a proper type.